packages/gatekeeper/src/helpers/utils.ts

```diff
@@ -7,3 +7,19 @@ export type ErrorCallback = (e: string) => string | void;
 export function errorCallback(error: string) {
   throw new Error(error);
 }
+
+/**
+ * Open another window and navigate to the logout URL.
+ * @param {string} logoutURL URL string representing the auth server logout URL endpoint.
+ * This function takes the approach of opening a new window and navigating to the logout
+ * url of the authentication server in order to go around the browser's CORS policy.
+ */
+export function logoutFromAuthServer(logoutURL: string) {
+  const logoutWindow: Window | null = window.open(logoutURL);
+  const timer: NodeJS.Timeout = setInterval(() => {
+    if (logoutWindow) {
+      logoutWindow.close();
+    }
+    clearInterval(timer);
+  }, 20);
```

Comment:
Let us move this int to the constants module https://github.com/onaio/js-tools/blob/master/packages/gatekeeper/src/helpers/constants.ts

Reply:
Basically I want users to be able to change this value to not always be 20.  Can we also make it an optional input, with a default, for this function?